ddtrace/logger.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+
+def configure_ddtrace_logger():
+    debug_log_level = os.environ.get("DD_TRACE_DEBUG", "false").lower() in ("true", "1")
```

Comment:
you might want to change to `debug_enabled` or something more "boolean sounding", since "debug_log_level" implies a level like "WARN", "DEBUG", "INFO", etc

Reply:
Renamed to `debug_enabled` ! I wasn't thinking `debug_log_level` could be used for a WARN log, but I see what you mean about an `enabled` sounding variable. I'll mark this as resolved.